### ISSUE TITLE
Sending blocks to Stoa failed on live system

### ISSUE THREAD

**Issue by Geod24** [bosagora/stoa#165]
```
Oct 30 03:56:53 eu-002 docker[9687]: [Stoa] 2020-10-30T03:56:53.221Z info Connected to Agora, block height is 0
Oct 30 03:56:53 eu-002 docker[9687]: [Stoa] 2020-10-30T03:56:53.223Z info Listening to requests on: 0.0.0.0:2828
Oct 30 04:00:49 eu-002 docker[9687]: [Stoa] 2020-10-30T04:00:49.975Z info Recovered a block with block height of 0
Oct 30 04:00:49 eu-002 docker[9687]: (node:38) UnhandledPromiseRejectionWarning: Error: SQLITE_ERROR: cannot rollback - no transaction is active
Oct 30 04:00:49 eu-002 docker[9687]: (Use `node --trace-warnings ...` to show where the warning was created)
Oct 30 04:00:49 eu-002 docker[9687]: (node:38) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
Oct 30 04:00:49 eu-002 docker[9687]: (node:38) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Looks like a serialization error which might be solved once #163 is merged.

**Comment by TrustHenry** [comment]
While the agora block height is 3 and stoa is performing the recover block operation, there seems to be an error while saving the block of 0 height.

**Comment by Geod24** [comment]
Now the error message is better:
```
Oct 30 07:28:03 eu-002 docker[8956]: (node:38) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 6)
Oct 30 07:28:03 eu-002 docker[8956]: (node:38) UnhandledPromiseRejectionWarning: Error: Validation failed: TxInputs - should be string
Oct 30 07:28:03 eu-002 docker[8956]:     at Function.isValidOtherwiseThrow (/stoa/bin/src/modules/data/validator/index.ts:95:23)
Oct 30 07:28:03 eu-002 docker[8956]:     at TxInputs.parseJSON (/stoa/bin/src/modules/data/TxInputs.ts:73:19)
Oct 30 07:28:03 eu-002 docker[8956]:     at Transaction.parseJSON (/stoa/bin/src/modules/data/Transaction.ts:87:47)
Oct 30 07:28:03 eu-002 docker[8956]:     at Block.parseJSON (/stoa/bin/src/modules/data/Block.ts:77:47)
Oct 30 07:28:03 eu-002 docker[8956]:     at Function.fromJSON (/stoa/bin/src/modules/data/Block.ts:92:30)
Oct 30 07:28:03 eu-002 docker[8956]:     at /stoa/bin/src/Stoa.ts:400:67
```

**Comment by MichaelKim20** [comment]
The cause of this problem is:
In the input member of transaction, it is index that is not string. But it is replaced by text. The reason was because of UINT64.
See) src/modules/data/types/TxInputs.d.ts
I think the cause of the error is probably a problem with the substitution of the number into the string.

**Comment by Geod24** [comment]
Then isn't this a duplicate of #168 ?

**Comment by MichaelKim20** [comment]
I don't think it's duplicated. `replace(/([\[:])?(\d+)([,\}\]])/g, "$1\"$2\"$3")` must be removed, sample JSON modified, and JSON Schema modified.
Even if #168 is complete, this part needs to be modified.
Live testing will be possible after #168 is complete.

**Comment by TrustHenry** [comment]
It was deployed to TestNet and the test results work well.

**Comment by MichaelKim20** [comment]
I checked the log below in the live system.
```
[Stoa] 2020-11-10T06:36:18.411Z info Agora endpoint: http://agora_node2:2826
[Stoa] 2020-11-10T06:36:18.414Z info sqlite3 database filename: /stoa/wd/data/database.db
[Stoa] 2020-11-10T06:36:18.446Z info Connected to Agora, block height is 0
[Stoa] 2020-11-10T06:36:18.449Z info Listening to requests on: 0.0.0.0:2828
[Stoa] 2020-11-10T06:36:21.533Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:36:25.722Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:36:31.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:36:35.723Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:36:41.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:36:45.727Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:36:51.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:36:55.720Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:37:01.494Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:37:05.719Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:37:11.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:37:15.720Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:37:21.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:37:25.720Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:37:31.493Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:37:35.719Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:37:41.495Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:37:45.722Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:37:51.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:37:55.722Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:38:01.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:38:05.721Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:38:11.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:38:15.719Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:38:21.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:38:25.722Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:38:31.490Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:38:35.721Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:38:41.493Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:38:45.722Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:38:51.514Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:38:55.720Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:39:01.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:39:05.720Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:39:11.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x7b78dd8534d0b964, distance : 6
[Stoa] 2020-11-10T06:39:15.719Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbb10e5050a2d0acd, distance : 6
[Stoa] 2020-11-10T06:39:17.304Z info Recovered a block with block height of 0
[Stoa] 2020-11-10T06:39:17.438Z info Saved a block with block height of 1
[Stoa] 2020-11-10T06:39:17.439Z info Ignored a block with block height of 1
[Stoa] 2020-11-10T06:39:20.664Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x9cbdcec817e1821e, distance : 7
[Stoa] 2020-11-10T06:39:20.916Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x076a1b2a3734e595, distance : 7
[Stoa] 2020-11-10T06:39:21.616Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x9cbdcec817e1821e, distance : 7
[Stoa] 2020-11-10T06:39:21.794Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x6c2fa7186aaa6c64, distance : 7
[Stoa] 2020-11-10T06:39:21.833Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x6c2fa7186aaa6c64, distance : 7
[Stoa] 2020-11-10T06:39:21.909Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x076a1b2a3734e595, distance : 7
[Stoa] 2020-11-10T06:39:22.495Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xd8c46934da947452, distance : 7
[Stoa] 2020-11-10T06:39:22.496Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xd8c46934da947452, distance : 7
[Stoa] 2020-11-10T06:39:23.888Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x256d1f020e79ee8b, distance : 7
[Stoa] 2020-11-10T06:39:23.889Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x256d1f020e79ee8b, distance : 7
[Stoa] 2020-11-10T06:39:31.524Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xfbf6b7057994d9ca, distance : 7
[Stoa] 2020-11-10T06:39:31.525Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x6c2fa7186aaa6c64, distance : 7
[Stoa] 2020-11-10T06:39:31.526Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xfbf6b7057994d9ca, distance : 7
[Stoa] 2020-11-10T06:39:41.287Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xfbf6b7057994d9ca, distance : 7
[Stoa] 2020-11-10T06:39:41.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x6c2fa7186aaa6c64, distance : 7
[Stoa] 2020-11-10T06:39:51.288Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xfbf6b7057994d9ca, distance : 7
[Stoa] 2020-11-10T06:39:51.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x6c2fa7186aaa6c64, distance : 7
[Stoa] 2020-11-10T06:40:01.287Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xfbf6b7057994d9ca, distance : 7
[Stoa] 2020-11-10T06:40:01.489Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x6c2fa7186aaa6c64, distance : 7
[Stoa] 2020-11-10T06:40:02.938Z info Saved a block with block height of 2
[Stoa] 2020-11-10T06:40:02.939Z info Ignored a block with block height of 2
[Stoa] 2020-11-10T06:40:03.830Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x06b6c3ab3ac45dd3, distance : 8
[Stoa] 2020-11-10T06:40:03.831Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x06b6c3ab3ac45dd3, distance : 8
[Stoa] 2020-11-10T06:40:10.649Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x8ab23532845304b2, distance : 8
[Stoa] 2020-11-10T06:40:10.650Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x8ab23532845304b2, distance : 8
[Stoa] 2020-11-10T06:40:11.151Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0xb6d44253987ab80c, distance : 8
[Stoa] 2020-11-10T06:40:11.152Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0xb6d44253987ab80c, distance : 8
[Stoa] 2020-11-10T06:40:11.529Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xb28742adc8de1087, distance : 8
[Stoa] 2020-11-10T06:40:11.531Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xb28742adc8de1087, distance : 8
[Stoa] 2020-11-10T06:40:11.729Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x58bc869bee498afc, distance : 8
[Stoa] 2020-11-10T06:40:11.731Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x58bc869bee498afc, distance : 8
[Stoa] 2020-11-10T06:40:12.473Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xe37ddd3bf5a5b7b5, distance : 8
[Stoa] 2020-11-10T06:40:12.474Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xe37ddd3bf5a5b7b5, distance : 8
[Stoa] 2020-11-10T06:40:21.287Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xb28742adc8de1087, distance : 8
[Stoa] 2020-11-10T06:40:21.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x58bc869bee498afc, distance : 8
[Stoa] 2020-11-10T06:40:31.286Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xb28742adc8de1087, distance : 8
[Stoa] 2020-11-10T06:40:31.493Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x58bc869bee498afc, distance : 8
[Stoa] 2020-11-10T06:40:41.286Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xb28742adc8de1087, distance : 8
[Stoa] 2020-11-10T06:40:41.493Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x58bc869bee498afc, distance : 8
[Stoa] 2020-11-10T06:40:48.477Z info Saved a block with block height of 3
[Stoa] 2020-11-10T06:40:48.478Z info Ignored a block with block height of 3
[Stoa] 2020-11-10T06:40:50.727Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0xff9abdcd5472690e, distance : 9
[Stoa] 2020-11-10T06:40:50.728Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0xff9abdcd5472690e, distance : 9
[Stoa] 2020-11-10T06:40:51.420Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0xd50b71784cad3883, distance : 9
[Stoa] 2020-11-10T06:40:51.424Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0xd50b71784cad3883, distance : 9
[Stoa] 2020-11-10T06:40:51.578Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x3e5a8d2bd620df6e, distance : 9
[Stoa] 2020-11-10T06:40:51.579Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x3e5a8d2bd620df6e, distance : 9
[Stoa] 2020-11-10T06:40:52.048Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x85bb716531de74af, distance : 9
[Stoa] 2020-11-10T06:40:52.050Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x85bb716531de74af, distance : 9
[Stoa] 2020-11-10T06:40:52.575Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xc931cf38ce39ca36, distance : 9
[Stoa] 2020-11-10T06:40:52.576Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xc931cf38ce39ca36, distance : 9
[Stoa] 2020-11-10T06:40:53.840Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0xb55beb9070b59b6e, distance : 9
[Stoa] 2020-11-10T06:40:53.841Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0xb55beb9070b59b6e, distance : 9
[Stoa] 2020-11-10T06:41:01.285Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x3e5a8d2bd620df6e, distance : 9
[Stoa] 2020-11-10T06:41:01.489Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x85bb716531de74af, distance : 9
[Stoa] 2020-11-10T06:41:11.287Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x3e5a8d2bd620df6e, distance : 9
[Stoa] 2020-11-10T06:41:11.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x85bb716531de74af, distance : 9
[Stoa] 2020-11-10T06:41:21.288Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x3e5a8d2bd620df6e, distance : 9
[Stoa] 2020-11-10T06:41:21.490Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x85bb716531de74af, distance : 9
[Stoa] 2020-11-10T06:41:31.285Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x3e5a8d2bd620df6e, distance : 9
[Stoa] 2020-11-10T06:41:31.502Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x85bb716531de74af, distance : 9
[Stoa] 2020-11-10T06:41:34.639Z info Saved a block with block height of 4
[Stoa] 2020-11-10T06:41:34.640Z info Ignored a block with block height of 4
[Stoa] 2020-11-10T06:41:40.684Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x120a196ab45819a4, distance : 10
[Stoa] 2020-11-10T06:41:40.685Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x120a196ab45819a4, distance : 10
[Stoa] 2020-11-10T06:41:41.011Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x78da3cec5cf9b5a8, distance : 10
[Stoa] 2020-11-10T06:41:41.012Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x78da3cec5cf9b5a8, distance : 10
[Stoa] 2020-11-10T06:41:41.527Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x2b53c6fade1e654c, distance : 10
[Stoa] 2020-11-10T06:41:41.528Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x2b53c6fade1e654c, distance : 10
[Stoa] 2020-11-10T06:41:41.756Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0xc5bc12b9cdcdb605, distance : 10
[Stoa] 2020-11-10T06:41:41.757Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0xc5bc12b9cdcdb605, distance : 10
[Stoa] 2020-11-10T06:41:42.516Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0x4f37745dda4ca325, distance : 10
[Stoa] 2020-11-10T06:41:42.517Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0x4f37745dda4ca325, distance : 10
[Stoa] 2020-11-10T06:41:43.859Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x07fb71c756caf446, distance : 10
[Stoa] 2020-11-10T06:41:43.860Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x07fb71c756caf446, distance : 10
[Stoa] 2020-11-10T06:41:51.284Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x2b53c6fade1e654c, distance : 10
[Stoa] 2020-11-10T06:41:51.490Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0xc5bc12b9cdcdb605, distance : 10
[Stoa] 2020-11-10T06:42:01.286Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x2b53c6fade1e654c, distance : 10
[Stoa] 2020-11-10T06:42:01.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0xc5bc12b9cdcdb605, distance : 10
[Stoa] 2020-11-10T06:42:11.285Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x2b53c6fade1e654c, distance : 10
[Stoa] 2020-11-10T06:42:11.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0xc5bc12b9cdcdb605, distance : 10
[Stoa] 2020-11-10T06:42:20.007Z info Saved a block with block height of 5
[Stoa] 2020-11-10T06:42:20.008Z info Ignored a block with block height of 5
[Stoa] 2020-11-10T06:42:20.780Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0xfd66e789b89cad9f, distance : 11
[Stoa] 2020-11-10T06:42:20.781Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0xfd66e789b89cad9f, distance : 11
[Stoa] 2020-11-10T06:42:21.133Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x58675965de640c95, distance : 11
[Stoa] 2020-11-10T06:42:21.134Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x58675965de640c95, distance : 11
[Stoa] 2020-11-10T06:42:21.469Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x033d75ea3cf5f2d3, distance : 11
[Stoa] 2020-11-10T06:42:21.470Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x033d75ea3cf5f2d3, distance : 11
[Stoa] 2020-11-10T06:42:21.720Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x1480060179804245, distance : 11
[Stoa] 2020-11-10T06:42:21.721Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x1480060179804245, distance : 11
[Stoa] 2020-11-10T06:42:22.411Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xa8a7834d0a17e9eb, distance : 11
[Stoa] 2020-11-10T06:42:22.412Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0xa8a7834d0a17e9eb, distance : 11
[Stoa] 2020-11-10T06:42:23.999Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x8eb8a4d53cdc2f16, distance : 11
[Stoa] 2020-11-10T06:42:24.000Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x8eb8a4d53cdc2f16, distance : 11
[Stoa] 2020-11-10T06:42:31.286Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x033d75ea3cf5f2d3, distance : 11
[Stoa] 2020-11-10T06:42:31.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x1480060179804245, distance : 11
[Stoa] 2020-11-10T06:42:41.286Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x033d75ea3cf5f2d3, distance : 11
[Stoa] 2020-11-10T06:42:41.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x1480060179804245, distance : 11
[Stoa] 2020-11-10T06:42:51.284Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x033d75ea3cf5f2d3, distance : 11
[Stoa] 2020-11-10T06:42:51.490Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x1480060179804245, distance : 11
[Stoa] 2020-11-10T06:43:01.287Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0x033d75ea3cf5f2d3, distance : 11
[Stoa] 2020-11-10T06:43:01.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x1480060179804245, distance : 11
[Stoa] 2020-11-10T06:43:06.280Z info Saved a block with block height of 6
[Stoa] 2020-11-10T06:43:06.281Z info Ignored a block with block height of 6
[Stoa] 2020-11-10T06:43:10.832Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x5c286e8210a26baa, distance : 12
[Stoa] 2020-11-10T06:43:10.833Z info Saved a pre-image enroll_key : 0x8c1561a4475df42a, hash : 0x5c286e8210a26baa, distance : 12
[Stoa] 2020-11-10T06:43:11.050Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x833158e253e4423f, distance : 12
[Stoa] 2020-11-10T06:43:11.050Z info Saved a pre-image enroll_key : 0x46883e83778481d6, hash : 0x833158e253e4423f, distance : 12
[Stoa] 2020-11-10T06:43:11.467Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbd481eb27057f71b, distance : 12
[Stoa] 2020-11-10T06:43:11.479Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbd481eb27057f71b, distance : 12
[Stoa] 2020-11-10T06:43:11.767Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x0e81b52e7cc0d2dd, distance : 12
[Stoa] 2020-11-10T06:43:11.768Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x0e81b52e7cc0d2dd, distance : 12
[Stoa] 2020-11-10T06:43:12.404Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0x08b3092b25b9401e, distance : 12
[Stoa] 2020-11-10T06:43:12.405Z info Saved a pre-image enroll_key : 0x94908ec79866cf54, hash : 0x08b3092b25b9401e, distance : 12
[Stoa] 2020-11-10T06:43:13.930Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x6eb905ed7f4f5660, distance : 12
[Stoa] 2020-11-10T06:43:13.931Z info Saved a pre-image enroll_key : 0x4dde806d2e09367f, hash : 0x6eb905ed7f4f5660, distance : 12
[Stoa] 2020-11-10T06:43:21.286Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbd481eb27057f71b, distance : 12
[Stoa] 2020-11-10T06:43:21.492Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x0e81b52e7cc0d2dd, distance : 12
[Stoa] 2020-11-10T06:43:31.284Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbd481eb27057f71b, distance : 12
[Stoa] 2020-11-10T06:43:31.490Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x0e81b52e7cc0d2dd, distance : 12
[Stoa] 2020-11-10T06:43:41.285Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbd481eb27057f71b, distance : 12
[Stoa] 2020-11-10T06:43:41.491Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x0e81b52e7cc0d2dd, distance : 12
[Stoa] 2020-11-10T06:43:51.285Z info Saved a pre-image enroll_key : 0xdb3931bd87d2cea0, hash : 0xbd481eb27057f71b, distance : 12
[Stoa] 2020-11-10T06:43:51.493Z info Saved a pre-image enroll_key : 0xb20da9cfbda971f3, hash : 0x0e81b52e7cc0d2dd, distance : 12
[Stoa] 2020-11-10T06:43:53.150Z info Saved a block with block height of 7
[Stoa] 2020-11-10T06:43:53.151Z info Ignored a block with block height of 7
```
